clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -315,14 +315,14 @@ public Void call() {
                         notificationChannel.setDescription(channelDescription);
                         notificationChannel.setShowBadge(showBadge);
                         notificationManager.createNotificationChannel(notificationChannel);
-                        instance.getConfigLogger().info(instance.getAccountId(),
+                        Logger.info(instance.getAccountId(),
                                 "Notification channel " + channelName.toString() + " has been created");
```

Comment:
@Anush-Shand One last change - In all the public methods that help create a Notification Channel in Android in this class, add one more logging line after this one - `Logger.info(instance.getAccountId(),"Notification channel " + channelName.toString() + " has been created");`
Add this as a debug log -
`Logger.debug(instance.getAccountId(),"Notification channel " + channelName.toString() + " has been created with Channel ID = " + channelId.toString()");`

This way we or the support teams can get the Channel ID without having to contact the marketing teams once we have the debug enabled APKs